src/sdk/odp/client/tabular_storage_client.py

```diff
@@ -333,7 +306,7 @@ def select_as_list(
             OdpResourceNotFoundError: If the schema cannot be found
         """
 
-        return list(self.select(resource_dto, filter_query, limit, cursor))
+        return list(self.select(resource_dto, filter_query, limit))
```

Comment:
why remove cursor here?

Reply:
`cursor` is reduntant in `select_as_list`. Removed.